### DISCUSSION TITLE
Open Garden Day T-Shirt Graphic

### DISCUSSION THREAD

**Discussion by SamuelWitke** [nbkyn-parks-community/nbkyn-parks-community.github.io#7] [Polls]
![Screenshot 2022-04-20 113505](https://user-images.githubusercontent.com/22154417/164268611-3d18793d-22a1-476c-88be-0a6717daf6fd.png)
![Screenshot 2022-04-20 113324](https://user-images.githubusercontent.com/22154417/164268625-a823a14f-3502-46aa-8328-5a8d413187fe.png)
![Screenshot 2022-04-20 113208](https://user-images.githubusercontent.com/22154417/164268639-e77bd24f-8573-483b-9afb-727f02cbf011.png)

**Comment by Marywitke** [comment]
Yellow with green ink

**Comment by SamuelWitke** [comment]
![thumbnail_scene-6](https://user-images.githubusercontent.com/22154417/169538201-20129563-5554-46b1-b9ca-fd67e1e756dc.png)
